packages/css-library/tokens/color.json

```diff
@@ -38,5 +38,84 @@
       "warm-light": { "value": "#e4e2e0"},
       "cool-light": { "value": "#dce4ef"}
     }
+  },
+  "v3-color": {
```

Comment:
All these are the [new available colors from USWDS v3 ](https://designsystem.digital.gov/design-tokens/color/theme-tokens/) and the [Tokens from these colors](https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/settings/_settings-color.scss#L86)